src/github.com/getlantern/flashlight/logging/useragents.go

```diff
@@ -0,0 +1,50 @@
+package logging
+
+import (
+	"bytes"
+	"fmt"
+	"regexp"
+	"sync"
+)
+
+type agentsMap map[string]int
```

Comment:
I tend to use typedefs for long-ish definitions to clarify what they actually mean. In this case I agree it doesn't help much.